<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_ModifyCodeAsset</class>
 <widget class="QDialog" name="Dialog_ModifyCodeAsset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>223</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>577</width>
    <height>223</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>577</width>
    <height>223</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LaTeX Office Writer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="3" colspan="6">
    <widget class="QLineEdit" name="lineEditCaption"/>
   </item>
   <item row="8" column="0" colspan="9">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Discard|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QSpinBox" name="spinBoxFirstLine">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QLabel" name="labelLastLine">
     <property name="text">
      <string>Last Line</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelFirstLine">
     <property name="text">
      <string>First Line</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelCaption">
     <property name="text">
      <string>Caption</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelSourceCode">
     <property name="text">
      <string>Source Code File</string>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <widget class="QSpinBox" name="spinBoxLastLine">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QRadioButton" name="radioButtonTop">
     <property name="text">
      <string>Top Caption</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="5">
    <widget class="QLineEdit" name="lineEditSourceCode">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLineEdit" name="lineEditLanguage">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3">
    <widget class="QRadioButton" name="radioButtonBottom">
     <property name="text">
      <string>Bottom Caption</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditSourceCode</tabstop>
  <tabstop>lineEditLanguage</tabstop>
  <tabstop>spinBoxFirstLine</tabstop>
  <tabstop>spinBoxLastLine</tabstop>
  <tabstop>lineEditCaption</tabstop>
  <tabstop>radioButtonBottom</tabstop>
  <tabstop>radioButtonTop</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_ModifyCodeAsset</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_ModifyCodeAsset</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
